src/audio/selector/selector.c

```diff
@@ -40,9 +40,16 @@ static const struct comp_driver comp_selector;
 
 LOG_MODULE_REGISTER(selector, CONFIG_SOF_LOG_LEVEL);
 
+#if CONFIG_IPC_MAJOR_3
```

Comment:
@RanderWang while you're doing this, can you also convert to use the module API as well?

Reply:
sure. It is on my next plan list